hack/test/test-wavefront-metrics.sh

```diff
@@ -66,11 +106,13 @@ function wait_for_query_non_zero() {
 function print_usage_and_exit() {
   echo "Failure: $1"
   echo "Usage: $0 [flags] [options]"
-  echo -e "\t-c wavefront instance name (default: 'nimba')"
   echo -e "\t-t wavefront token (required)"
-  echo -e "\t-n config cluster name for metric grouping (default: \$(whoami)-<default version from file>-release-test)"
-  echo -e "\t-v collector version (default: load from 'release/VERSION')"
-  echo -e "\t-e name of a file containing any extra asserts that should be made as part of this test"
+  echo -e "\t-n config cluster name for metric grouping (required)"
+  echo -e "\t-c wavefront instance name (default: 'nimba')"
+  echo -e "\t-c collector version (default: load from 'release/COLLECTOR_VERSION')"
```

Comment:
Did you mean to duplicate the -c option?

Reply:
Thank you! Fixed